app/src/main/java/org/simple/clinic/sync/SyncWorker.kt

```diff
@@ -78,10 +78,11 @@ class SyncWorker(context: Context, workerParams: WorkerParameters) : Worker(cont
             facilitySync.sync()
         ))
         .doOnError { e ->
-          if (e !is IOException) {
-            Timber.e(e)
-            crashReporter.report(e)
+          val resolvedError = ErrorResolver.resolve(e)
+          if (resolvedError.shouldBeReported) {
+            crashReporter.report(resolvedError.actualCause)
           }
+          Timber.e(e)
```

Comment:
Since we report Timber logs to Sentry, won't this continue to get reported even if the error was a network error?

Reply:
Timber logs are only forwarded to Sentry as breadcrumbs. They won't show up as individual crash reports.